dbms/src/Storages/KVStore/MultiRaft/RegionCFDataBase.h

```diff
@@ -24,7 +24,7 @@ namespace DB
 
 struct TiKVRangeKey;
 using RegionRange = RegionRangeKeys::RegionRange;
-using RegionDataRes = size_t;
+using RegionDataRes = int64_t;
```

Comment:
why this is changed to int64_t but the return type of `RegionData::insert` is not changed?

Reply:
And why we need a signed value here?

Reply:
The lock cf tend to change. If this happen, a new lock can have smaller size.

Reply:
fixed